samples/Templates/Scenarios/FoodOrder.template.json

```diff
@@ -1,174 +1,155 @@
-                        "wrap": true
-                    },
-                    {
-                        "type": "Input.ChoiceSet",
-                        "id": "ChickenAllergy",
-                        "style": "expanded",
-                        "isMultiSelect": true,
-                        "choices": [
-                            {
-                                "title": "I'm allergic to peanuts",
-                                "value": "peanut"
-                            }
-                        ]
-                    },
-                    {
-                        "type": "Input.Text",
-                        "id": "ChickenOther",
-                        "isMultiline": true,
-                        "placeholder": "Any other preparation requests?"
-                    }
-                ],
-                "actions": [
-                    {
-                        "type": "Action.Submit",
-                        "title": "OK",
-                        "data": {
-                            "FoodChoice": "Chicken"
-                        }
-                    }
-                ],
-                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
-            }
-        },
-        {
-            "type": "Action.ShowCard",
-            "title": "${hasMenu.hasMenuSection[0].hasMenuSection[1].name}",
-            "card": {
-                "type": "AdaptiveCard",
-                "body": [
-                    {
-                        "type": "TextBlock",
-                        "text": "${hasMenu.hasMenuSection[0].hasMenuSection[1].description}",
-                        "size": "Medium",
-                        "wrap": true
-                    },
-                    {
-                        "type": "Input.Toggle",
-                        "id": "Vegetarian",
-                        "title": "Please prepare it vegan",
-                        "valueOn": "vegan",
-                        "valueOff": "notVegan",
-                        "wrap": false
-                    },
-                    {
-                        "type": "Input.Text",
-                        "id": "VegOther",
-                        "isMultiline": true,
-                        "placeholder": "Any other preparation requests?"
-                    }
-                ],
-                "actions": [
-                    {
-                        "type": "Action.Submit",
-                        "title": "OK",
-                        "data": {
-                            "FoodChoice": "Vegetarian"
-                        }
-                    }
-                ],
-                "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
-            }
-        }
-    ]
+	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+	"type": "AdaptiveCard",
+	"version": "1.3",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "Your registration is almost complete",
+			"size": "Medium",
```

Comment:
other files have changed words like Medium and Bolder for their lowercase versions, for consistency reasons we should keep them the same across files

Reply:
@almedina-ms Good call out on consistency. Do we want to keep the uppercase version or lowercase version?

Reply:
@almedina-ms 
I created another PR with the enum lowercase changes - https://github.com/microsoft/AdaptiveCards/pull/8029
I'll go ahead and resolve the other comments, but keep this one open for tracking :)